Fixes the following error after running volk_modtool.
ImportError: cannot import name 'volk_modtool' from partially initialized module 'volk_modtool' (most likely due to a circular import)

--- a/cmake/Modules/VolkPython.cmake
+++ b/cmake/Modules/VolkPython.cmake
@@ -243,7 +243,7 @@
 
             #on windows, python files need an extension to execute
             get_filename_component(pyfile_ext ${pyfile} EXT)
-            if(WIN32 AND NOT pyfile_ext)
+            if(MSVC AND NOT pyfile_ext)
                 set(pyfile_name "${pyfile_name}.py")
             endif()
 
